tools/mixmodel4repeated_measures/mixmodel.xml

```diff
@@ -0,0 +1,268 @@
+<tool id="mixmodel" name="mixmodel" version="3.1.0">
+    <description>ANOVA for repeated measures statistics</description>
+
+    <requirements>
+        <requirement type="package">r-lme4</requirement>

```

Comment:
a version is missing here

Reply:
Do you mean a version for the package?

Reply:
Mh, ok, not ideal but workable.

Reply:
In the end I tried specifying the version and I did not run into issue so I added it through 6c74258